src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -39,23 +40,29 @@
 @Api(tags = "Actors")
 public class ActorsController {
 
-    @Autowired
-    BuildConfig buildConfig;
-
     @Autowired
     private ActorsRepository actorRepository;
 
     @RequestMapping(value = "", method = RequestMethod.GET)
     @ApiOperation(value = "Get all actors", notes = "Retrieve and return all actors")
     @ApiResponses(value = { @ApiResponse(code = 200, message = "List of actor objects") })    
-public Flux<Actor> finActors(ServerHttpResponse response) {
+    public Mono<Actor> findByActorId(ServerHttpResponse response) {
+        return actorRepository.findByActorId("nm0000502");
+    }
 
-    //return actorRepository.findAll();
-    return actorRepository.findByActorId("nm0000502");
-    
-//   response.setStatusCode(HttpStatus.OK);
-//   return Flux.just(buildConfig.getBuildVersion());
-}
 
+    // @RequestMapping(value = "/{id}", method = RequestMethod.GET)
+    // @ApiOperation(value = "Get single actor", notes = "Retrieve and return a single actor by actor ID")
+    // @ApiResponses(value = {
+    //         @ApiResponse(code = 200, message = "The actor object"),
+    //         @ApiResponse(code = 404, message = "An actor with the specified ID was not found") })
+    // public ResponseEntity<Actor> getActor(@ApiParam(value = "The ID of the actor to look for", example = "nm0000002", required = true) @PathVariable("id")  String actorId) {
+    //     Mono<Actor> actor = actorRepository.findByActorId(actorId);
+    //     if (actor.isPresent()) {
```

Comment:
This looks like it has the param needed above.  Let's fix one of them and delete unnecessary code.

Reply:
This was fixed earlier but did not send all the changes over. You can see again and it should be fixed.